source/_components/sensor.sensehat.markdown

```diff
@@ -34,56 +35,13 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
+  
 
-### {% linkable_title Installation instruction for the All-In-One installer and HASSbian %}
+#### Customizing the Sense HAT data
```

Comment:
The previous issue has a styling issue when merge occurred. This corrects that on top of that PR. This is the correct version.